### ISSUE TITLE
Mac OS X: Cannot open port (PortInUseException)

### ISSUE THREAD

**Issue by nholbert** [NetLogo/GoGo-RXTX-Extension#2]
Error when trying to open the port on a macbook for a usb gogo board using NetLogo 4.1.3 and 5.0RC4

``` text
Extension exception: Could not open port /dev/tty.usbmodem411 : Port is already in use: gnu.io.PortInUseException: Unknown Application
error while observer running GOGO:OPEN
  called by procedure SETUP
  called by Button 'setup'
```

**Comment by SethTisue** [comment]
ticket in main repo: https://github.com/NetLogo/NetLogo/issues/68

**Comment by SethTisue** [comment]
fixed, see 26221bb79b6804198e6c741de6c4c09d29860470 and subsequent commits
